llvm/projects/evm-tests/common_tests/storage.cpp

```diff
@@ -0,0 +1,69 @@
+#include "evm_sdk.h"
+#include "storage_gen.h"
+
+using namespace evm;
+
+static constexpr const char* STORAGE_YAML = R"(
```

Comment:
It is not used in C++ yet, but there are plans to run the generator at the preprocessor phase. It can be a comment, but IMO the comment doesn't look very solid.